UPGRADE-2.0.md

```diff
@@ -0,0 +1,143 @@
+# Upgrade guide from 1.x to 2.x
+
+## Needed actions
+This is the list of actions that you need to take when upgrading this bundle from the 1.x to the 2.x version:
+
+- Remove `FixturesTrait`:
+    - Access through the service `liip_test_fixtures.services.database_tool_collection` instead
+    - Use `loadAliceFixture(…)` instead of `loadFixtureFiles(…)`
+    - `loadFixtures()` and `loadFixtureFiles()` only accept 2 arguments, here are the old arguments and the new way:
+        - 3rd argument `$omName`: call `$databaseTool->setObjectManagerName($omName);` instead
+        - 4th argument `$registryName`: call `$databaseTool->get(null, $registryName);` instead
+        - 5th argument `$purgeMode`: call `$databaseTool->setPurgeMode($purgeMode);` instead
+- Deprecated the `@DisableDatabaseCache` annotation, use `$this->databaseTool->setDatabaseCacheEnabled(false);` instead,
+  and don't forget to 
```

Comment:
reset it `$this->databaseTool->setDatabaseCacheEnabled(false);` at the end of the test.